### PR TITLE
fix: use `userId`

### PR DESCRIPTION
**Pull request by omBratteng**

### CODE REVIEW

**Review by capJavert — APPROVED** [review]

File: src/common/mailing.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const syncSubscription = async function (
         );
       if (!digest) {
         await manager.getRepository(UserPersonalizedDigest).delete({
-          id: customer.id,
+          userId: customer.id,
```
Comment (capJavert) [inline src/common/mailing.ts:117]: damn did we delete more than we should have?

Comment (rebelchris) [reply src/common/mailing.ts:117]: id doesn't exist :)  (so should be ok)

Comment (omBratteng) [reply src/common/mailing.ts:117]: Nah, just got error. That row doesn't exist anyways. Though, strange that linter didn't catch that the table doesn't have `id` column
 
> Property "id" was not found in "UserPersonalizedDigest". Make sure your query is correct.

Comment (omBratteng) [reply src/common/mailing.ts:117]: That would've been bad stupid from their side :triggered:

Comment (capJavert) [reply src/common/mailing.ts:117]: I have to speed up that `strict` mode update.. it would have caught this

Comment (omBratteng) [reply src/common/mailing.ts:117]: Not sure... this is the type definition
image

Comment (capJavert) [reply src/common/mailing.ts:117]: [strict](https://www.typescriptlang.org/tsconfig/#strict) mode in typescript yes

Comment (capJavert) [reply src/common/mailing.ts:117]: But even then it would not since their internal `ObjectId` always has `id` property 🤦

Comment (omBratteng) [reply src/common/mailing.ts:117]: Yeah 😿

**Pull request merged**